source/unified-test-format/unified-test-format.rst

```diff
@@ -3894,6 +3894,9 @@ Changelog
 ..
   Please note schema version bumps in changelog entries where applicable.
 
+:2022-05-10: **Schema version 1.14.**
+             Add ``ignoreLogMessages`` and ``ignoreExtraLogs`` fields
+             to ``expectLogMessages``section.
```

Comment:
```suggestion
             to ``expectedLogMessagesForClient``.
```

This is consistent with what you actually changed in `schema-1.14.json`.

Also, you forgot to add documentation for these fields in the spec itself. Please add that to the [`expectedLogMessagesForClient`](https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#expectedlogmessagesforclient) section.

Reply:
Thanks, done.